src/main/kotlin/com/vk/admstorm/AdmStormStartupActivity.kt

```diff
@@ -41,8 +42,12 @@ class AdmStormStartupActivity : ProjectActivity {
         }
 
         //TODO: remove it after release AdmStorm on Windows
-        if (SystemInfo.isWindows) {
-            AdmErrorNotification("AdmStorm is not available on Windows yet").show()
+        if (!SystemInfo.isWindows) {
```

Comment:
Why not?

Reply:
I pushed it while I was testing it :(
Thank you!